code/src/index.css

```diff
@@ -1,13 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
 body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: "Roboto", sans-serif;
+  background-color: rgb(250, 247, 233);
+  color: rgb(55, 27, 16);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  font-weight: 300;
+  font-size: 16px;
+  width: 300px;
+
+}
+
+p {
+  font-family: "Roboto";
+}
+
+.mainBody {
+  width: 300px;
+}
+
+.submitBox {
+  display: flex;
+  margin-top: 20px;
+  border: 1px solid gold;
+  background-color: cornsilk;
+  padding: 5px 70px 10px 10px;
+  box-shadow: 5px 5px gold;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+textarea {
+  width: 260px;
+  height: 80px;
+  border: none;
+  margin-bottom: -10px;
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 14px;
+  color: rgb(55, 27, 16);
+  outline-color: goldenrod;
+}
+
+button {
+  color: rgb(55, 27, 16);
+  border-radius: 50px;
+  border: none;
+  background-color: gold;
+  font-family: "Roboto";
+  padding: 8px 20px 8px 20px;
+  box-shadow: 2px 2px goldenrod;
+}
+.counter {
+  font-size: 12px;
+  color: rgba(55, 27, 16, 0.7);
+  text-align: right;
+}
+
+.counter-red {
+  color: red;
+  text-align: right;
+  font-size: 12px;
+}
+
+.errorAlert {
+  font-size: 12px;
+  margin-bottom: 10px;
+}
+
+.listSection {
+  margin-top: 20px;
+}
+
+.listBox {
+  border: 1px solid gold;
+  margin-bottom: 16px;
+  box-shadow: 5px 5px gold;
+  background-color: white;
+  border-radius: 5px 15px;
+}
+
+.heartBtn {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: rgb(55, 27, 16);
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(250, 247, 233);
+  padding: 5px 7px;
+  box-shadow: none;
+  margin-right: 5px;
+  margin-bottom: 10px;
+}
+
+.likedBtn {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: rgb(55, 27, 16);
+  border-radius: 50%;
+  border: none;
+  background-color: gold;
+  padding: 5px 7px;
+  box-shadow: none;
+  margin-right: 5px;
+  margin-bottom: 10px;
+}
+
+.thought {
+  margin-left: 10px;
+  font-family: 'Courier New', Courier, monospace;
+  font-weight: 500;
+}
```

Comment:
How and where in you code are you making the text of this card to not overflow if it where an textsnippet that was to long? In the inspect there is a "overflow-wrap: anywhere;"  but i cannot find it and how you did it. Is it magic? Curios to know!